tox.ini

```diff
@@ -47,9 +47,11 @@ deps =
     django50: Django>=5.0,<5.1
     wagtail52: wagtail>=5.2,<6.0
     wagtail60: wagtail>=6.0,<6.1
+    wagtail61: wagtail>=6.1,<6.2
+    wagtail62: wagtail>=6.2,<6.3
     interactive: wagtail>=5.2
 
-install_command = python -Im pip install -U {opts} {packages}
+install_command = python -Im pip install {opts} {packages}
```

Comment:
The `-U` flag here was causing Wagtail to be upgraded to the latest version in [the install_package step](https://github.com/torchbox/wagtail-grapple/actions/runs/13176328127/job/36776414261?pr=415#step:6:21). You can see `wagtail==6.4` in [the output of `pip freeze`](https://github.com/torchbox/wagtail-grapple/actions/runs/13176328127/job/36776414261?pr=415#step:6:23).

Reply:
you can remove this since the default is `python -I -m pip install {opts} {packages}`

Reply:
Ah, grand! This means I can use `tox-uv` locally, to install environments faster 🚀

Reply:
adding `tox-uv` was on my list to try with grapple, so go wild ;)